pkgs/data/icons/nordzy-icon-theme/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, bash
+, gtk3
+, nordzy-themes ? [ "all" ] # Override this to only install selected themes
+}:
+
+let
+  themes-arg-string = lib.strings.concatMapStrings (theme: "-t ${theme} ") nordzy-themes;
+in
+stdenv.mkDerivation rec {
+  pname = "nordzy-icon-theme";
+  version = "5c247a4f19cf9849615631d1bf77727b945b634e";
```

Comment:
Maybe this is toooooo long for `version`?
Shall we move this to `rev` and just set `unstable-yyyy-mm-dd` or follow [RFC 107](https://github.com/NixOS/rfcs/blob/083116c0be0315977d7852b02b3f54c1fb766e62/rfcs/0107-version-normalization.md) for `version`?